joomla/joomla-cms
# [5.2] Fix increment on non-alphanumeric string deprecation

Pull Request for Issue #43896 .

### Summary of Changes
Fix increment on non-alphanumeric string deprecation on php 8.3.


### Testing Instructions
Enable Error Reporting to Maximum.
Create/save an article.
Create/save a banner.
Create/save a contact.
Create/save a newsfeed.
Create/save a tag.
See PHP error log.
Apply PR.
Under Publishing tab, check Revision value incrementing.

### Actual result BEFORE applying this Pull Request
PHP Deprecated: Increment on non-alphanumeric string is deprecated


### Expected result AFTER applying this Pull Request
No error messages.
